### PR TITLE
Add Ipe.app v1.0

### PR DESCRIPTION
**Pull request by miccal**
This is my first attempt at contributing, so if I have done anything wrong I apologise in advance.

### CODE REVIEW

File: Casks/Ipe.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'ipe' do
+  version '7.1.10'
+  sha256 '5e14f13d213905af34561a0d169e9f1f9eed91ae8f85f163df951af2c2392198'
+
+  url 'https://dl.bintray.com/otfried/generic/ipe/7.1/ipe-7.1.10-mac.dmg'
+  name 'Ipe'
+  homepage 'http://ipe.otfried.org'
+  license :oss    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/Ipe.rb]: `:gpl`. Also, remove the comment (read it).

File: Casks/Ipe.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'ipe' do
+  version '7.1.10'
+  sha256 '5e14f13d213905af34561a0d169e9f1f9eed91ae8f85f163df951af2c2392198'
+
+  url 'https://dl.bintray.com/otfried/generic/ipe/7.1/ipe-7.1.10-mac.dmg'
```
Comment (vitorgalvao) [inline Casks/Ipe.rb]: Needs a `# bintray.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/Ipe.rb]: Also, line should be `url "https://dl.bintray.com/otfried/generic/ipe/#{version.to_f}/ipe-#{version}-mac.dmg"` for easier future updates.

**Comment by miccal** [comment]
Done, thanks vitorgalvao.

File: Casks/ipe.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'ipe' do
+  version '7.1.10'
+  sha256 '5e14f13d213905af34561a0d169e9f1f9eed91ae8f85f163df951af2c2392198'
+
+  # bintray.com is the official download host per the vendor homepage
+  url "https://dl.bintray.com/otfried/generic/ipe/#{version.to_f}/ipe-#{version}-mac.dmg"
+  name 'Ipe'
+  homepage 'http://ipe.otfried.org'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/ipe.rb]: Still needs to be changed to `:gpl`.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Pull request closed without merging**